exercisefiles/node/README.md

```diff
@@ -4,7 +4,7 @@ Demo project for running labs to evaluate Copilot viability
 
 > Make sure GitHub Copilot is configure and enabled for the current language, just check the status bar on the bottom right corner of VS Code.
 
-## Instructions
+### Exercise 1: Introduction
```

Comment:
Changes here make sense, the introduction now is more of an introduction than previously when the whole task was laid out. I would say this is fine also.